tests/generator/test_ethernets.py

```diff
@@ -164,6 +164,27 @@ def test_eth_match_by_mac_rename(self):
         self.assert_nm(None)
         self.assert_nm_udev(NM_UNMANAGED % 'lom1' + NM_UNMANAGED_MAC % '11:22:33:44:55:66')
 
+    def test_eth_match_by_pciid_rename(self):
```

Comment:
nitpick: Thanks for providing some test cases! I'd love to see an integration test along the lines of `tests/integration/ethernets.py:test_rename_interfaces()`, too (if possible).